specification/cognitiveservices/OpenAI.Inference/models/chat.completions.tsp

```diff
@@ -0,0 +1,197 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "./completions.common.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+@doc("A description of the intended purpose of a message within a chat completions interaction.")
+enum ChatRole {
+    @doc("The role that instructs or sets the behavior of the assistant.")
+    @projectedName("json", "system")
+    system,
```

Comment:
Good catch! I think I'll do it the other way around for the time being, if it doesn't run afoul of a rule-become-warning; `@projectedName` doesn't actually appear to work here yet, so capitalizing the enum values here would generate underlying strings like `System` instead of `system` (and cause things to break). That also applies to `CompletionsFinishReason`, but those values included much bigger renames rather than just a casing issue.